alerter.c

```diff
@@ -19,6 +29,7 @@ void alertInCelcius(float farenheit) {
         // let us keep a count of failures to report
         // However, this code doesn't count failures!
         // Add a test below to catch this bug. Alter the stub above, if needed.
+        assert(Alert == 0);

```

Comment:
In your code, `Alert` is a flag that you want to be `1` when there was an alert. Go ahead and assert that it should be `1`.  Move the assert to `main`, so the test and the code remains separate.